src/shell.ts

```diff
@@ -570,7 +582,7 @@ async function executeCommandArgs(commandArgs: string[], context: Context) {
   try {
     // ignore the result of writing to stdin because it may
     // have not finished before the process finished
-    const _ignore = writeStdin(context.stdin, p, completeSignal);
+    const _ignore = writeStdin(context.stdin, p, completeSignal); // TODO does this have to be assigned then ?
```

Comment:
It doesn''t have to be, but it's a floating promise. The `_ignore` is done to signify "yes, I know this is floating. Don't add an await here"